pkg/controller/clusteringress/clusteringress_controller.go

```diff
@@ -108,7 +108,7 @@ func (r *ReconcileClusterIngress) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
-	return reconcile.Result{}, nil
+	return reconcile.Result{Requeue: true}, nil
```

Comment:
Why do we set `Requeue: true` here? If we get to this point, doesn't it mean the reconcile succeeded and we have no need to requeue?

Reply:
Setting ```Requeue: true``` helps  to make sure the desired state.

When i added https://github.com/openshift-knative/knative-openshift-ingress/pull/6/files#diff-d9edc9c5da0d95247db22dbd278e58e7R89
this condition in testcase ```Reconcile``` result becomes **true** or else it was **false** always.

So that's why to make sure added that piece.

Reply:
But why are we testing that Requeue is true? Requeing the request means it will go through the reconcile loop again - that does not seem desirable, and may in fact result in an infinite loop?

Or am I misunderstanding how it's being used?

Reply:
Yes you are right Requeue true will call Reconile again.

I  misunderstood by seeing 
https://github.com/operator-framework/operator-sdk-samples/blob/master/memcached-operator/pkg/controller/memcached/memcached_controller.go#L133